### PR TITLE
Adds new fonts to Storybook config and website

### PR DESCRIPTION
**Pull request by TheSonOfThomp**
## ✍️ Proposed changes

🎟 _Jira ticket:_ [Name of ticket](https://jira.mongodb.org/browse/[name-of-ticket])

## 🛠 Types of changes

- [ ] Tooling (updates to workspace config or internal tooling)
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] I have run `yarn changeset` and documented my changes

- [ ] I have added my new package to the global tsconfig
- [ ] I have added my new package to the Table of Contents on the global README

### CODE REVIEW

File: website/styles/globals.tsx [hunk 1]
```diff
@@ -48,8 +48,173 @@ export const globalStyles = css`
...
+        format('woff'),
+      url('${fontsURL}/value-serif/MongoDBValueSerif-Bold.woff2')
+        format('woff2'),
+      url('${fontsURL}/value-serif/MongoDBValueSerif-Bold.ttf')
+        format('truetype');
+  }
+
   html {
-    font-family: 'Akzidenz', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+    font-family: 'Euclid', 'Akzidenz', 'Helvetica Neue', Helvetica, Arial,
```
Comment (DesignerDave) [inline website/styles/globals.tsx:216]: I don't think we should flip this switch yet.

File: storybook/preview-head.html [hunk 2]
```diff
@@ -1,11 +1,14 @@
...
-<link href="https://fonts.googleapis.com/css2?family=Source+Code+Pro:ital,wght@0,400;0,600;1,400&display=swap" rel="stylesheet" />
+<link
+  href="https://fonts.googleapis.com/css2?family=Source+Code+Pro:ital,wght@0,400;0,600;1,400&display=swap"
+  rel="stylesheet"
+/>
 
 <style type="text/css">
   html {
-    font-family: 'Akzidenz', Helvetica, Arial, sans-serif;
+    font-family: 'Euclid', 'Akzidenz', Helvetica, Arial, sans-serif;
```
Comment (DesignerDave) [inline storybook/preview-head.html:11]: Are we able to do this selectively? Not all components are going to be able to use this yet.

**Review by DesignerDave — CHANGES_REQUESTED** [review]
Couple notes. Also, we do need to add the new font family to our tokens package.

File: storybook/preview-head.html [hunk 3]
```diff
@@ -74,4 +77,173 @@
...
     /* Legacy iOS */
   }
+
+  /**
+   * Euclid
+   */
+
+  /* Normal */
+  @font-face {
+    font-family: 'Euclid';
```
Comment (TheSonOfThomp) [inline storybook/preview-head.html:87]: Call this "Euclid Circular A"

File: storybook/preview-head.html [hunk 4]
```diff
@@ -74,4 +77,173 @@
...
+    font-style: italic;
+  }
+
+  /**
+   * Value Serif
+   */
+
+  /* Normal */
+  @font-face {
+    font-family: 'Value';
```
Comment (TheSonOfThomp) [inline storybook/preview-head.html:216]: Call this "MongoDB Value Serif"

File: website/styles/globals.tsx [hunk 5]
```diff
@@ -48,6 +48,170 @@ export const globalStyles = css`
...
+        format('truetype');
+  }
+
+  /**
+   * Value Serif
+   */
+
+  /* Normal */
+  @font-face {
+    font-family: 'Value Serif';
```
Comment (TheSonOfThomp) [inline website/styles/globals.tsx:181]: Call this "MongoDB Value Serif"

File: website/styles/globals.tsx [hunk 6]
```diff
@@ -48,6 +48,170 @@ export const globalStyles = css`
     /* Legacy iOS */
   }
 
+  /**
+   * Euclid
+   */
+
+  /* Normal */
+  @font-face {
+    font-family: 'Euclid';
```
Comment (TheSonOfThomp) [inline website/styles/globals.tsx:57]: Call this "Euclid Circular A"

File: storybook/preview-head.html [hunk 7]
```diff
@@ -74,4 +77,173 @@
...
+  /* Bold */
+  @font-face {
+    font-family: 'Euclid';
+    src: url('https://d2va9gm4j17fy9.cloudfront.net/fonts/euclid-circular/EuclidCircularA-Bold-WebXL.woff')
+        format('woff'),
+      url('https://d2va9gm4j17fy9.cloudfront.net/fonts/euclid-circular/EuclidCircularA-Bold-WebXL.woff2')
+        format('woff2'),
+      url('https://d2va9gm4j17fy9.cloudfront.net/fonts/euclid-circular/EuclidCircularA-Bold.ttf')
+        format('truetype');
+    font-weight: 700, bold;
```
Comment (TheSonOfThomp) [inline storybook/preview-head.html:119]: Remove "bold"

File: storybook/preview-head.html [hunk 8]
```diff
@@ -74,4 +77,173 @@
...
+  /* Semibold */
+  @font-face {
+    font-family: 'Euclid';
+    src: url('https://d2va9gm4j17fy9.cloudfront.net/fonts/euclid-circular/EuclidCircularA-Semibold-WebXL.woff')
+        format('woff'),
+      url('https://d2va9gm4j17fy9.cloudfront.net/fonts/euclid-circular/EuclidCircularA-Semibold-WebXL.woff2')
+        format('woff2'),
+      url('https://d2va9gm4j17fy9.cloudfront.net/fonts/euclid-circular/EuclidCircularA-Semibold.ttf')
+        format('truetype');
+    font-weight: 600;
```
Comment (TheSonOfThomp) [inline storybook/preview-head.html:194]: This is "bold"

File: storybook/preview-head.html [hunk 9]
```diff
@@ -74,4 +77,173 @@
...
+  /* Light */
+  @font-face {
+    font-family: 'Euclid';
+    src: url('https://d2va9gm4j17fy9.cloudfront.net/fonts/euclid-circular/EuclidCircularA-Light-WebXL.woff')
+        format('woff'),
+      url('https://d2va9gm4j17fy9.cloudfront.net/fonts/euclid-circular/EuclidCircularA-Light-WebXL.woff2')
+        format('woff2'),
+      url('https://d2va9gm4j17fy9.cloudfront.net/fonts/euclid-circular/EuclidCircularA-Light.ttf')
+        format('truetype');
+    font-weight: 100, 200, 300;
```
Comment (TheSonOfThomp) [inline storybook/preview-head.html:144]: Just call this `300`

**Review by DesignerDave — APPROVED** [review]

File: storybook/preview-head.html [hunk 10]
```diff
@@ -74,4 +77,178 @@
...
+  /* Semibold */
+  @font-face {
+    font-family: 'Euclid Circular A';
+    src: url('https://d2va9gm4j17fy9.cloudfront.net/fonts/euclid-circular/EuclidCircularA-Semibold-WebXL.woff')
+        format('woff'),
+      url('https://d2va9gm4j17fy9.cloudfront.net/fonts/euclid-circular/EuclidCircularA-Semibold-WebXL.woff2')
+        format('woff2'),
+      url('https://d2va9gm4j17fy9.cloudfront.net/fonts/euclid-circular/EuclidCircularA-Semibold.ttf')
+        format('truetype');
+    font-weight: 700;
```
Comment (DesignerDave) [inline storybook/preview-head.html:120]: We should be setting this as the bold font weight I believe as well.

**Pull request merged**